src/js/script.js

```diff
@@ -37,6 +37,28 @@ export const NETWORKS = [
 export const API_BASE_URL = 'https://api.checker.network'
 export const SPARK_API_BASE_URL = 'https://stats.filspark.com'
 
+/**
+ * Converts a given Date object to a local date string in ISO format (YYYY-MM-DD).
+ *
+ * @param {Date} d - The Date object to be converted.
+ * @returns {string} The local date as a string in ISO format.
+ */
+const getLocalDayAsISOString = (d) => {
+  return [
+    d.getFullYear(),
+    String(d.getMonth() + 1).padStart(2, '0'),
+    String(d.getDate()).padStart(2, '0')
+  ].join('-')
+}
+
+/**
+ * Calculates a date string in ISO format for a given number of days ago.
+ *
+ * @param {number} daysAgo - The number of days ago from the current date.
+ * @returns {string} The ISO string representation of the calculated date.
+ */
+export const getDateDaysAgo = (daysAgo) => getLocalDayAsISOString(new Date(Date.now() - daysAgo * 24 * 60 * 60 * 1000))
```

Comment:
I assume these two functions were copy pasted, and therefore don't need review

Reply:
Yes, exactly.

Reply:
Thanks for confirming!